src/screens/Home/CreateScreen/VideoLoadingScreen.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import AnimatedEllipsis from 'react-native-animated-ellipsis';
+import {
+  Text,
+  View,
+} from 'react-native'
+import styles from './styles'
+
+export default function CreateScreen() {
```

Comment:
Can you rename this screen `RenderingVideoScreen` in 4 places
1. Rename the folder it's in from CreateScreen to `RenderingVideoScreen`
2. Rename filename 
3. Rename function name 
4. Update CreateScreen references in MainTabNavigator to RenderingVideoScreen